src/lib/support/DefaultStorageKeyAllocator.h

```diff
@@ -69,6 +69,10 @@ class DefaultStorageKeyAllocator
         return Format("a/%" PRIx16 "/%" PRIx32 "/%" PRIx32, aPath.mEndpointId, aPath.mClusterId, aPath.mAttributeId);
     }
 
+    // Binding Table
```

Comment:
The comment does not seem to help me understand the code: the method names already have a 'BindingTable' prefix.